internal/webserver/wsauthconfig/config.go

```diff
@@ -0,0 +1,46 @@
+package wsauthconfig
```

Comment:
Can extend the existing tests for the config and also add validation to the individual parameters?

Reply:
you can find examples in the other configs and here https://github.com/go-playground/validator

Reply:
I added config validation and testing in the latest commit